### PR TITLE
Changes applied

### PR DESCRIPTION
**Pull request by Meril99**
Changs applied

### CODE REVIEW

File: Assignment4/MiangouilaMeril99/task07.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        ?entity foaf:knows ?known2 .
+        FILTER (?known1 != ?known2)
+    }
+    GROUP BY ?entity
+    HAVING (COUNT(DISTINCT ?known1) >= 2)
+"""
+
+for r in g.query(q5):
+    print(r)
+
```
Comment (dgarijo) [inline Assignment4/MiangouilaMeril99/task07.py:110]: All these exercises are correct, nicely done.
Now you need to do the alternative using just rdflib and not SPARQL. We have seen it in class too

**Review by dgarijo — CHANGES_REQUESTED** [review]
You almost have it all. You are just missing the rdflib exercises (have a look at the notebooks we saw in class)

File: Assignment4/MiangouilaMeril99/task07.py (hunk 1 above)
Comment (Meril99) [reply Assignment4/MiangouilaMeril99/task07.py:110]: Ok so task 06 and task 07 with only rdflib. Can I put it in the same file ?

Comment (dgarijo) [reply Assignment4/MiangouilaMeril99/task07.py:110]: Tasks 7.1 ... 7.5 should be done with rdflib too. Task 06 is already with rdflib. Just add it after each exercise.

**Pull request closed without merging**